controllers/articleController.js

```diff
@@ -17,4 +19,18 @@ const fetchArticles = (req, res, next) => {
     .catch(next)
 };
 
-module.exports = {fetchArticleById, fetchArticles}
\ No newline at end of file
+const fetchCommentsByArticleId = (req, res, next) => {
+    const {article_id} = req.params
+    const promises = [selectCommentsByArticleId(article_id)]
+
+    if(article_id){
+        promises.push(selectArticleById(article_id))
+    }
```

Comment:
There is no need to have an "if" statement here since this controller function will only ever get invoked id the endpoint is met i.e. there is a parameter, even if the ID is "not-valid", it still exists.